src/protocol-parser/protocol-parser.ts

```diff
@@ -978,6 +1078,9 @@ export namespace Session {
     Log.EventNames.LogEntryAddedEvent,
     CDP.AllEvents,
     CDP.EventNames.EventReceivedEvent,
+    Network.AllEvents,
+    Network.EventNames.BeforeRequestSentEvent,
+    Network.EventNames.ResponseCompletedEvent,
```

Comment:
In the other file (`protocol.ts`), we have just `Network.EventNames` instead of listing each individual event. Can we do the same thing here? (same for the other event types, while we're here - http://go/tott/524)

Reply:
> In the other file (`protocol.ts`), we have just `Network.EventNames` instead of listing each individual event. Can we do the same thing here? (same for the other event types, while we're here - http://go/tott/524)

Out of scope of the change.